sfa_dash/templates/forms/form_macros.jinja

```diff
@@ -17,7 +17,15 @@
   {% endif %}
   <label>{{ label }}</label><br/>
   <div class="input-wrapper">
-    <input type="{{ type }}" class="form-control {{ class }} {{ name }}-field" {{ extra_attr }}  name="{{ name }}"  {% if name == 'name' %}{{ name_validation() }}{% endif %} {% if form_data is not none %}value="{{ form_data.get(name, default) }}"{% endif %}>{% if units is not none %}{{ units }}{% endif %}
+    <input type="{{ type }}"
+     class="form-control {{ class }} {{ name }}-field"
+     {{ extra_attr }}
+     name="{{ name }}"
+     {% if name == 'name' %}{{ name_validation() }}{% endif %}
+     {% if form_data is not none %}
+     value="{% autoescape true %}{{ form_data.get(name, default) }}{% endautoescape %}"
```

Comment:
Can we just escape the variable? https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping